src/Features/Core/Diagnostics/EngineV1/DiagnosticIncrementalAnalyzer_GetDiagnostics.cs

```diff
@@ -493,12 +513,43 @@ protected override void FilterDiagnostics(AnalysisData analysisData, Func<Diagno
 
         private class IDELatestDiagnosticGetter : LatestDiagnosticsGetter
         {
-            public IDELatestDiagnosticGetter(DiagnosticIncrementalAnalyzer owner) : this(owner, null)
+            private readonly bool _concurrent;
```

Comment:
since you added field to the base one, why not move this to base as well? and pass down concurrent flag to the base?